libs/langchain/langchain/chat_models/bedrock.py

```diff
@@ -26,6 +27,8 @@ def convert_messages_to_prompt(
     ) -> str:
         if provider == "anthropic":
             prompt = convert_messages_to_prompt_anthropic(messages=messages)
```

Comment:
@baskaryan @WouterDurnez If using a bedrock anthropic model this always resolves to NotImplementedError due to incorrect implementation of If statement. This should be elseif or immediately return the prompt value once match with anthropic. This is currently an breaking change.

Reply:
🙈 https://github.com/langchain-ai/langchain/pull/13684 --> added a fix

Reply:
🙈 Sorry, missed that. Added a fix but then saw the PR above. Thx!